### ISSUE TITLE
Make EIP155 signatures compatible with tester

### ISSUE THREAD

**Issue by pcppcp** [raiden-network/raiden-libs#44]
## What's wrong

`eth-tester` has some support for EIP155 that activates replay protection for transactions, but it still doesn't work reliably (i.e. `getTransactionByHash` still raises errors)

## How to fix it
- short term: do not use EIP155 with tests, monkeypatch local signing method to use old-style `v in (0, 1, 27 ,28)` 
- long term: pester `eth-tester` team to have full support of signatures with `v >= 35`

**Comment by palango** [comment]
This should be fixed with https://github.com/ethereum/eth-tester/pull/82

**Comment by pcppcp** [comment]
yes, I've seen that PR. But it's just a workaround and it can take some time until the change propagates to `master` and pypi. Correct `v` validation needs a network id and this would require more changes to the `validation.outbound` module.

I can patch module's `validate_signature_v` however, this could be a better solution than replacing signature algo.
